website/docs/d/dns_managed_zone.html.markdown

```diff
@@ -32,7 +32,9 @@ resource "google_dns_record_set" "dns" {
 
 ## Argument Reference
 
-* `name` - (Required) A unique name for the resource.
+* `name` - (Optional) A unique name for the resource. Required if `dns_name` is not set.
+
+* `dns_name` - (Optional) The fully qualified DNS name of this zone, e.g. `terraform.io.`.  Required if `name` is not set.
```

Comment:
Should this mention that if multiple zones have the same DNS name, it will use the first?